airbyte-integrations/connectors/source-mssql/src/test-integration/java/io/airbyte/integrations/source/mssql/AbstractSshMssqlSourceAcceptanceTest.java

```diff
@@ -88,14 +91,16 @@ private static Database getDatabaseFromConfig(final JsonNode config) {
             String.format(DatabaseDriver.MSSQLSERVER.getUrlFormatString(),
                 config.get(JdbcUtils.HOST_KEY).asText(),
                 config.get(JdbcUtils.PORT_KEY).asInt(),
-                config.get(JdbcUtils.DATABASE_KEY).asText()),
+                config.get(JdbcUtils.DATABASE_KEY).asText()) + ";encrypt=false",
```

Comment:
I don't feel strongly about this but IIRC there's a `create` method which also takes a map of connection parameters, you could pass `Map.of("encrypt", "false")` to it.